Gruntfile.js

```diff
@@ -509,6 +512,9 @@ module.exports = function (grunt) {
     // task: build dist/ for browser
     grunt.registerTask('build-browser', ['build', 'uglify']);
 
+    // task: build dist/ for S3 (gzipped)
+    grunt.registerTask('build-s3', ['build', 'uglify', 'compress']);
```

Comment:
Maybe rename this to `build-browser-compressed`, since it's not really doing anything with s3 in this task--that's just how we use it.